source/css/sass/_mixins--spacing.scss

```diff
@@ -21,6 +21,26 @@
   }
 }
 
+// Fallbacks for CSS logical properties contained within this mixin require the following treatment of HTML dir attributes:
+//  - document level: always specified, via the HTML element
+//  - block level: specified on every element within a block describing a direction switch.
+//
+// For example:
+// <html lang="..." dir="ltr">
+// ...
+// <div>
+//   Doesn't need a dir attribute. Most cases will be like this.
+// </div>
+//
+//<div class="test" dir="rtl">
+//
+//  This block changes the text direction. Every descendant element must have its own dir attribute....
```

Comment:
'Every descendant <ins>block-level</ins> element'?

Reply:
I think it needs to be every element: left/right spacing will need to be able to change side depending on the reading direction,  and this could potentially apply to inline elements as well as block-level elements.

Reply:
Possibly. Let's see what happens.